fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryTypeFilterPlan.java

```diff
@@ -220,4 +221,14 @@ public PlannerGraph rewritePlannerGraph(@Nonnull List<? extends PlannerGraph> ch
                         ImmutableMap.of("types", Attribute.gml(getRecordTypes().stream().map(Attribute::gml).collect(ImmutableList.toImmutableList())))),
                 childGraphs);
     }
+
+    private static int stringHashUnordered(@Nonnull Iterable<String> strings) {
```

Comment:
Why is this copying over `PlanHashable.stringHashUnordered`? Is this because that method is now deprecated, and this is the only caller?

Reply:
yes -- this is implementation is bas as in that it is sorting the strings rather than making the hash commutative. I don't want other (future) callers to use this.